RetroBar/Utilities/Settings.cs

```diff
@@ -181,22 +196,22 @@ public bool AllowFontSmoothing
             }
         }
 
-        public AppBarEdge Edge
+        public int Edge
         {
             get
             {
                 if (settings.Edge >= 0 && settings.Edge <= 3)
                 {
-                    return (AppBarEdge)settings.Edge;
+                    return settings.Edge;
                 }
 
-                return AppBarEdge.Bottom;
+                return (int)AppBarEdge.Bottom;
             }
             set
             {
-                if (settings.Edge != (int)value)
+                if (settings.Edge != value)
```

Comment:
I noticed when changing language, the properties window briefly sets this to -1. This causes some visual jank when changing language if the user's edge is set to anything other than the fallback value (bottom). I think adding a bounds check here would be a good fix for this.

Reply:
Yes, you're right, I'll fix it in a moment.

Reply:
For some reason, it is restoring to 0, I will see what the problem is now...

Reply:
Having settings read from a static field of the properties UI feels like a really hacky way to resolve this. Any reason you didn't take the above approach?

Reply:
It is because when changing the context or content of the "Location" ComboBox, the previously selected value is cleared and reset (set to -1), so we need to know how to get the value before the change other than -1, this also gets the localized text with the updated value.
As for the limits adjustment or bounds check, I wouldn't know how to apply it in this case...

Reply:
My idea is replacing this line with `if (settings.Edge != value && value >= 0 && value <= 3)`. That way the -1 never gets saved, but the properties window still behaves properly.

Reply:
All right, it was that easy... I'll take that!